site/content/en/docs/for-users/faq.md

```diff
@@ -96,7 +89,8 @@ volumes:
 
 ## How to make unassigned tasks not visible to all users
 
-Set [reduce_task_visibility](../../settings/base.py#L424) variable to `True`.
+Set [reduce_task_visibility](https://github.com/openvinotoolkit/cvat/blob/develop/cvat/settings/base.py#L424)
+variable to `True`.
```

Comment:
@azhavoro 
Maybe add the information about the CVAT should be rebuilt after the change?